.github/workflows/canary-integration-test.yml

```diff
@@ -160,8 +167,14 @@ jobs:
 
     - name: wait for ceph to be ready
       run: |
-        tests/scripts/validate_cluster.sh osd
-        kubectl -n rook-ceph get pods
+        tests/scripts/validate_cluster.sh osd >> log.txt
```

Comment:
Instead of capturing everything from the test log, how about the following?
- The validate_cluster.sh script should dump the pod logs to individual files. For example, individual commands inside the `display_status()` func would dump to a file.
- The validate_cluster.sh script would take the test folder as an argument so the script knows where to write the files.

This way we still have the main test output in the console, but pod logs and other extra info is in the artifacts.

Reply:
Done.